### DISCUSSION TITLE
docker-compose deployment unable to access internet

### DISCUSSION THREAD

**Discussion by kpleines** [benbusby/whoogle-search#954] [General]
latest benbusby/whoogle-search docker image deployed using docker-compose crashed on startup on CentOs 9

* CentOs 9
* Docker version 23.0.0, build e92dd87
* docker-compose version 1.23.2, build 1110ad01

**To Reproduce**
1. start docker image using "docker-compose up -d"
2. image starts but the search engine don't comes up (cant connect using http)
3. "docker logs whoogle-search" search the exception

**Deployment Method**
- [ ] Heroku (one-click deploy)
- [x ] Docker
- [ ] `run` executable
- [ ] pip/pipx
- [ ] Other: [describe setup]

**Version of Whoogle Search**
- [ x] Latest build from [source] (i.e. GitHub, Docker Hub, pip, etc)
- [ ] Version [version number]
- [ ] Not sure

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Firefox ]
 - Version [109]
[docker-compose.yml.txt](https://github.com/benbusby/whoogle-search/files/10698932/docker-compose.yml.txt)

**Additional context**
[whoogle-error.log.txt](https://github.com/benbusby/whoogle-search/files/10699477/whoogle-error.log.txt)

**Comment by benbusby** [comment]
The error logs show that the container doesn't have internet access, or otherwise can't perform network requests, not Whoogle itself throwing an exception. There's probably a configuration issue, but nothing jumps out to me and I can't replicate the issue on my end. I'm going to convert this to a discussion in case someone else can chime in and help out.
